### ISSUE TITLE
download MODIS (crs: SR-ORG:6974) data appears obscured

### ISSUE THREAD

**Issue by cug-xyx** [google/Xee#119]
Dear Developers:

Thank you so much for being able to develop the awesome `xee`.

I was downloading the **MODIS** series product (**MODIS/061/MOD16A2GF**) and noticed that the downloaded data was shaded.

I think it's a problem with the `crs`.

Here are the images and code:

```
ic_name = 'MODIS/061/MOD16A2GF'

ic = ee.ImageCollection(ic_name) \
  .select("ET") \
  .filterDate('2013-06-02', '2013-06-03')

ds = xarray.open_dataset(
    ic, engine='ee',
    crs = 'EPSG:4326', 
    scale=0.25
)
ds
```
![image](https://github.com/google/Xee/assets/77427610/a9ad0178-956d-4da6-941f-6442e6ecd589)

```
ds.to_netcdf('test.nc')

r = xr.open_dataarray('test.nc')
r
```

![image](https://github.com/google/Xee/assets/77427610/02f7a804-dd81-4085-9a14-6ebd9c7e7764)

```
r \
    .assign_attrs(crs='EPSG:4326') \
    .transpose() \
    .plot(
    vmin=0, vmax=300,
    levels=[i for i in range(0, 270, 30)],
    colors=['#ffffff', '#fcd163', '#99b718', '#66a000',
            '#3e8601', '#207401', '#056201', '#004c00', '#011301']
)
```

![image](https://github.com/google/Xee/assets/77427610/a68c800f-d53f-42bf-8155-f3773a6dac4b)

**Comment by simonff** [comment]
What specifically do you mean by "shaded"?

**Comment by cug-xyx** [comment]
I'm sorry. Maybe I didn't describe it properly.

This image is ok on `GEE`.

```
var dataset = ee.ImageCollection('MODIS/NTSG/MOD16A2/105');
var ET = dataset.select('ET').filterDate('2013-06-02', '2013-06-03');
var ETVis = {
  min: 0,
  max: 300,
  palette:
      ['#ffffff', '#fcd163', '#99b718', '#66a000',
            '#3e8601', '#207401', '#056201', '#004c00', '#011301'],
};
Map.addLayer(ET, ETVis, 'Evapotranspiration');
```
![image](https://github.com/google/Xee/assets/77427610/0ba32a0e-8f04-463c-b38f-fe0904649784)

But exporting locally using `xee` results in something like the following:

![image](https://github.com/google/Xee/assets/77427610/81280cbd-2c30-4434-8933-540a44206f89)

**Comment by simonff** [comment]
Ah, you mean the weird yellow area over Americas and Antarctica. I agree this is wrong.

**Comment by cug-xyx** [comment]
Indeed. But `xee` is so good that I just can't live without it, so I'm looking for a solution here

**Comment by simonff** [comment]
Let's see what the actual developers say - I'm just another xee user

**Comment by scheip** [comment]
Have you tried this with different times or different bands and does it yield the same result? I am also having a problem with bringing MODIS data across via xee, though a bit different to what you are showing.

**Comment by cug-xyx** [comment]
> Have you tried this with different times or different bands and does it yield the same result? I am also having a problem with bringing MODIS data across via xee, though a bit different to what you are showing.

I've tried different periods and it's the same problem :(

**Comment by noahgolmant** [comment]
What I notice about this dataset is that each band has a different scale, which is the same case as #122. There is a chunk of code in [`get_info`](https://github.com/google/Xee/blob/f05e82b751d54433099c89c9ce7d68c629c592dc/xee/ext.py#L260) where xee infers a number of collection-wide attributes like the primary coordinate. Just spitballing, but maybe that has something to do with it? Like that data differs between bands with different scales?

**Comment by cug-xyx** [comment]
Hi @tylere,

I don't know what happed, but I think the bug has been fixed, here is the same example i reproduced.

https://colab.research.google.com/drive/1QL8-_NL_Y4vupzhdAvfpvlWD0IYf7OZX?usp=sharing

**Comment by tylere** [comment]
Hi @cug-xyx,

I'm not able to access the Colab link you shared. Could you either:

1. share your Colab notebook on Drive publicly (Share -> Anyone with the link), or
2. share the Colab notebook as a GitHub Gist (File -> Save a copy as a GitHub Gist)

**Comment by cug-xyx** [comment]
Hi @tylere 

I changed the permissions and now you should be able to access the link!

**Comment by tylere** [comment]
Thanks for updating the permissions.
I was able to reproduce your successful run, but only after I installed the netcdf4 package into Colab.
`!pip install --quiet netCDF4`

Script: https://colab.research.google.com/gist/tylere/f9ff7c75d5c9611dfa4192f003b02c33/xee-issue-119.ipynb#scrollTo=SrNyCSEKxnLy

Given that the original issue seems to have been resolved, I am closing the issue.
